morphic.js

```diff
@@ -5749,9 +5749,9 @@ SliderMorph.uber = CircleBoxMorph.prototype;
 
 function SliderMorph(start, stop, value, size, orientation, color) {
     this.init(
-        start || 1,
-        stop || 100,
-        value || 50,
+        typeof start === 'number' ? start : 1,
```

Comment:
That still wouldn't work because +0 || 1 === 1, and start will still default to 1 when when it should be 0.